syncserver/src/web/extractors.rs

```diff
@@ -134,7 +134,7 @@ impl From<BatchBsoBody> for PostCollectionBso {
 // This will pull the first accepted content type listed, or the highest rated non-accepted type.
 fn get_accepted(req: &HttpRequest, accepted: &[&str], default: &'static str) -> String {
     let mut candidates = Accept::parse(req).unwrap_or_else(|_| {
-        Accept(vec![qitem(
+        Accept(vec![QualityItem::max(
```

Comment:
I got this from Ethan's doc, but please double check it because I have no clue what's actually going on here.

Reply:
Oh, wow. Yeah, this is a fairly convoluted way to deal with a very convoluted field specification. Basically, this is constructing the `Accept` header's `;q=` value, set to the max `1.0` for the Mime qualifier for whatever is specifed as `default`. (e.g. for `txt` it should produce something like `Accept: plain/txt:q=1.0`)